crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -5,22 +5,46 @@
 def read_names():
```

Comment:
to make this more reusable pass in the path as a Path object

Reply:
As in "from pathlib import Path"; that type of Path?